staging/src/k8s.io/kubectl/pkg/drain/cordon.go

```diff
@@ -72,7 +72,7 @@ func (c *CordonHelper) UpdateIfRequired(desired bool) bool {
 // updating the given node object; it may return error if the object cannot be encoded as
 // JSON, or if either patch or update calls fail; it will also return a second error
 // whenever creating a patch has failed
-func (c *CordonHelper) PatchOrReplace(clientset kubernetes.Interface) (error, error) {
+func (c *CordonHelper) PatchOrReplace(clientset kubernetes.Interface, dryRun bool) (error, error) {
```

Comment:
It's not super clear what type of dry-run here? Probably doesn't matter.

Reply:
Renamed s/dryRun/serverDryRun for clarity